portifolio/contato.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Edivaldo - Contato</title>
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+</head>
+<body>
+    <div class="header">
+        <h1>Edivaldo Vaz - Front-End Student</h1> 
+    </div>
+    
+    <div class="topnav">
+        <a href="file:///C:/Users/ediva/Edivaldo-Lima/portifolio/index.html">Início</a>
```

Comment:
Utilize links relativos:
`<a href="index.html">Início</a>`
Dessa forma que você fez só funciona na sua máquina.
O mesmo vale pra todos os outros links que você fez!